certified-connectors/Conversica/apiDefinition.swagger.json

```diff
@@ -0,0 +1,193 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Conversica API",
```

Comment:
description should be 30 to 500 characters

Reply:
I have updated both the title and description.